src/Arcus.WebApi.Unit/OpenApi/OAuthAuthorizeController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Arcus.WebApi.Unit.OpenApi
+{
+    [ApiController]
+    public class OAuthAuthorizeController : ControllerBase
+    {
+        public const string AuthorizedRoute = "oauth/authorize",
+                            NoneAuthorizedRoute = "oauth/none";
+
+        [HttpGet]
+        [Route(AuthorizedRoute)]
+        [Authorize]
+        public Task<IActionResult> GetAuthorized(HttpRequestMessage request)
```

Comment:
Just return `IActionResult` since it's not doing anything async? 🤔 
```suggestion
        public IActionResult GetAuthorized(HttpRequestMessage request)
```